package.json

```diff
@@ -100,6 +100,12 @@
           "type": "boolean",
           "default": false,
           "description": "Enable/disable to Deploy the yaml resources on cluster"
+        },
+        "vs-tekton.StartPipeline": {
+          "Title": "Start Pipeline",
+          "type": "boolean",
+          "default": false,
+          "description": "Enable/disable to Start Pipeline in webView"
```

Comment:
`Enable/disable to Start Pipeline in webView` Is not clear as for me, 
maybe `Enable WebView Wizard to Start Pipeline` is better.
As for me `Enable/disable` is very confusing.